use/posts.js

```diff
@@ -110,11 +110,14 @@ export const usePosts = () => {
   const postFields = [
     {
       name: 'username',
+      label: t('posts.edit.label.username'),
```

Comment:
`$t('posts.edit.lable.username')`, not `t('posts.edit.lable.username')`
I mean you missed `$` as prefix.

Reply:
@best-lucky1030 `$t` is available in vue templates only, this is js file. 
You might have missed https://github.com/we-dance/platform/pull/93

Reply:
oh, got it. really missed #93